content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -26,11 +26,12 @@ The following methods exist for installing kubectl on Windows:
 
 1. Download the [latest release {{< param "fullversion" >}}](https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
 
-   Or if you have `curl` installed, use this command:
+   use this command:
```

Comment:
```suggestion
   Or if you have `curl` installed, use this command:
```
We could even specify that it's the default for all Windows 10 and 11 builds from December 2017.

Reply:
I tried it on 2 windows devices still curl not working , I think if we keep Invoke-WebRequest it helps a lot of users with windows background without forcing users to adopt to specific windows builds.